### PR TITLE
[BACK-2679] Resolve dependabot issues

### PR DESCRIPTION
**Pull request by darinkrauss**
- Resolve dependabot issues
- Update to node v20.11.0
- Update jshint
- Resolve lint issues
- Update scripts
- Update .travis.yml
- Add .nvmrc
- Bump module version
- https://tidepool.atlassian.net/browse/BACK-2679

### CODE REVIEW

**Review by toddkazakov — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,13 @@
+dist: jammy
+
 language: node_js
+
 node_js:
-  - "16.14.2"
-  - "stable"
+  - 20.10.0
```
Comment (jh-bate) [inline .travis.yml:6]: update to `20.11.0`

**Review by jh-bate — DISMISSED** [review]

File: .travis.yml (hunk 1 above)
Comment (darinkrauss) [reply .travis.yml:6]: Using latest LTS, v20.11.0.

Comment (darinkrauss) [reply .travis.yml:6]: Updated PR description.

**Review by jh-bate — APPROVED** [review]
LGTM

**Pull request merged**